services/airflow/src/dags/datenspende_vitaldata/post_processing/pivot_tables.py

```diff
@@ -0,0 +1,33 @@
+from database import (
+    create_db_context,
+    teardown_db_context,
+    execute_sql,
+    query_all_elements,
+)
+
+
+PIVOT_TARGETS = [(9, "steps", "int"), (65, "resting_heartrate", "int")]
+
+
+def pivot_vitaldata():
```

Comment:
Integration tests fail because once the task runs in an actual worker, it does not know about the environment variables you set in you tests (like the `TARGET_DB`). You are injecting them correctly via the execute dag method, which passes them via the dag context. However, you have to pick it up and set it as a environment variable again on your worker for the db_context factories to pick it up. 
`set_env_variable_from_dag_config_if_present` comes in handy here.